src/main/java/mate.academy/lib/Inject.java

```diff
@@ -1,5 +1,8 @@
 package mate.academy.lib;
 
-public @interface Inject {
+import java.lang.annotation.Retention;
+import java.lang.annotation.RetentionPolicy;
 
+@Retention(RetentionPolicy.RUNTIME)
+public @interface Inject {
```

Comment:
Consider adding the `@Target` annotation to specify where this annotation can be applied. According to the checklist, you should not forget to add `@Target` to custom annotations .